app/controllers/doorkeeper_applications_controller.rb

```diff
@@ -21,7 +21,18 @@ def update
 
   def users_with_access
     query = policy_scope(User).with_access_to_application(@application)
-    @users = query.page(params[:page]).per(100)
+    @users = query.page(params[:page]).per(10)
```

Comment:
Has this slipped in from another commit, or is there a reason we're cutting down on the number of users here? Might be worth updating the commit message if so!